test/Dialect/FIRRTL/imconstprop.mlir

```diff
@@ -551,3 +551,41 @@ firrtl.circuit "Issue3372"  {
     firrtl.strictconnect %zero, %c0_ui1 : !firrtl.uint<1>
   }
 }
+
+// -----
+
+// CHECK-LABEL: "SendThroughRef"
+firrtl.circuit "SendThroughRef" {
+  firrtl.module private @Bar(out %_a: !firrtl.ref<uint<1>>) {
+    %zero = firrtl.constant 0 : !firrtl.uint<1>
+    %ref_zero = firrtl.ref.send %zero : !firrtl.uint<1>
+    firrtl.strictconnect %_a, %ref_zero : !firrtl.ref<uint<1>>
+  }
+  // CHECK:  firrtl.strictconnect %a, %c0_ui1 : !firrtl.uint<1>
```

Comment:
I love it when a test boils down to a `CHECK-LABEL` and a lone `CHECK`. Super focused. 💯